code/modules/projectiles/ammunition/ammo_casings.dm

```diff
@@ -28,17 +28,37 @@
 /obj/item/ammo_casing/c9mmap
 	desc = "A 9mm bullet casing."
 	caliber = "9mm"
-	projectile_type =/obj/item/projectile/bullet/armourpiercing
+	projectile_type = "/obj/item/projectile/bullet/armourpiercing"
```

Comment:
Yyyyyup; it's horrible.

I figured it was just bad practice--but nope; the Saber SMG was firing normal bullets for all its magazine types without it...